.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: "Close stale issues"
+on:
+  schedule:
+    - cron: "0 0 * * *"
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue is stale because it has been open 14 days with no activity. Remove stale label or comment or this will be closed in 5 days'
+          close-issue-message: "This issue was closed because it has been inactive for 14 days since being marked as stale."
+          days-before-stale: 14
+          days-before-close: 5
+          any-of-labels: can't reproduce,help,needs clarification,needs repro script
```

Comment:
Remove "needs repro script". Right now, "needs repro script" means the issue is waiting on us because we may have enough info for a repro script, but haven't had time to try yet.

Reply:
```suggestion
          any-of-labels: can't reproduce,help,needs clarification
```